mobile/lib/ui/settings/backup/backup_status_screen.dart

```diff
@@ -20,12 +23,47 @@ class BackupStatusScreen extends StatefulWidget {
 
 class _BackupStatusScreenState extends State<BackupStatusScreen> {
   LinkedHashMap<String, BackupItem> items = FileUploader.instance.allBackups;
+  List<BackupItem>? result;
 
   @override
   void initState() {
     super.initState();
 
     checkBackupUpdatedEvent();
+    getAllFiles();
+  }
+
+  Future<void> getAllFiles() async {
+    result = (await SearchService.instance.getAllFiles())
+        .map(
+          (e) {
+            return BackupItem(
+              status: BackupItemStatus.uploaded,
+              file: e,
+              collectionID: e.collectionID ?? 0,
+              completer: null,
+            );
+          },
+        )
+        .toList()
```

Comment:
We need to ignore files that are not uploaded and also the one that are not owned by the user. Also, we can have multiple files with same uploadID.